tests/files/drupal9-mariadb-single/.lagoon.yml

```diff
@@ -1,6 +1,6 @@
 docker-compose-yaml: docker-compose.yml
 
-ssh: lagoon-core-ssh.lagoon.svc:2020
+ssh: lagoon-remote-ssh-portal.lagoon.svc.cluster.local:2222
```

Comment:
THis should be the ssh token service in core (either via existing ssh service or new ssh token one.

Reply:
Does drush only use this field to get a token?

Reply:
yes, that's my understanding